addons/l10n_ec/data/account_group_data.xml

```diff
@@ -0,0 +1,679 @@
+            <field name="code_prefix_start">520218</field>
+            <field name="name">AGUA, ENERGIA, LUZ, Y TELECOMUNICACIONES</field>
+            <field name="parent_id" ref="ec5202"/>
+        </record>
+
+        <record id="ec520220" model="account.group">
+            <field name="code_prefix_start">520220</field>
+            <field name="name">IMPUESTOS, CONTRIBUCIONES Y OTROS</field>
+            <field name="parent_id" ref="ec5202"/>
+        </record>
+
+        <record id="ec520221" model="account.group">
+            <field name="code_prefix_start">520221</field>
+            <field name="name">DEPRECIACIONES</field>
+            <field name="parent_id" ref="ec5202"/>
+        </record>
+
+        <record id="ec52022101" model="account.group">
+            <field name="code_prefix_start">52022101</field>
+            <field name="name">PROPIEDADES, PLANTA Y EQUIPO</field>
+            <field name="parent_id" ref="ec520221"/>
+        </record>
+
+        <record id="ec520222" model="account.group">
+            <field name="code_prefix_start">520222</field>
+            <field name="name">AMORTIZACIONES</field>
+            <field name="parent_id" ref="ec5202"/>
+        </record>
+
+        <record id="ec520223" model="account.group">
+            <field name="code_prefix_start">520223</field>
+            <field name="name">GASTO DETERIORO</field>
+            <field name="parent_id" ref="ec5202"/>
+        </record>
+
+        <record id="ec52022301" model="account.group">
+            <field name="code_prefix_start">52022301</field>
+            <field name="name">PROPIEDADES, PLANTA Y EQUIPO</field>
+            <field name="parent_id" ref="ec520223"/>
+        </record>
+
+        <record id="ec520228" model="account.group">
+            <field name="code_prefix_start">520228</field>
+            <field name="name">OTROS GASTOS</field>
+            <field name="parent_id" ref="ec5202"/>
+        </record>
+
+        <record id="ec52022818" model="account.group">
+            <field name="code_prefix_start">52022818</field>
+            <field name="name">GND GASTOS NO DEDUCIBLES</field>
+            <field name="parent_id" ref="ec520228"/>
+        </record>
+
+        <record id="ec5203" model="account.group">
+            <field name="code_prefix_start">5203</field>
+            <field name="name">GASTOS FINANCIEROS</field>
+            <field name="parent_id" ref="ec52"/>
+        </record>
+
+        <record id="ec5204" model="account.group">
+            <field name="code_prefix_start">5204</field>
+            <field name="name">OTROS GASTOS</field>
+            <field name="parent_id" ref="ec52"/>
+        </record>
+
+        <record id="ec520402" model="account.group">
+            <field name="code_prefix_start">520402</field>
+            <field name="name">OTROS</field>
+            <field name="parent_id" ref="ec5204"/>
+        </record>
+
+    </data>
+</odoo>
```

Comment:
@Stentor we should have an account group template instead.  Check if it already exisist.

Reply:
Solved in incoming commit